tests/event_loop_monitor.js

```diff
@@ -0,0 +1,72 @@
+import { Meteor } from 'meteor/meteor';
+import { EventLoopMonitor } from '../lib/event_loop_monitor';
+
+Tinytest.addAsync(
+  'EventLoopMonitor - basic usage',
+  function (test, done) {
+    const monitor = new EventLoopMonitor(100);
+    monitor.start();
+
+    // Saturate the event loop so that we can detect lag.
+    for ( let i = 0; i < 10000; i++ ) {
+      setTimeout(() => {}, 99);
+    }
+
+    Meteor.setTimeout(function () {
+      const status = monitor.status();
+
+      console.log(process.version);
```

Comment:
There are a few console.log's in this file. Are they still needed?

Reply:
I will do a few more test runs, and once I feel they are completely safe I will remove it.